manifests/params.pp

```diff
@@ -122,7 +133,7 @@
   # Owner of the environments dir: for cases external service needs write
   # access to manage it.
   $server_environments_owner   = $user
-  $server_environments_group   = 'root'
+  $server_environments_group   = $root_group
```

Comment:
"Administrators" seems a good choice according to https://support.microsoft.com/kb/243330